scripts/release.sh

```diff
@@ -5,12 +5,15 @@ set -o nounset
 set -o pipefail
 
 readonly REPO_ROOT="${REPO_ROOT:-$(git rev-parse --show-toplevel)}"
+CR_INDEX_DIR=.cr-index
+CR_RELEASE_PACKAGES_DIR=.cr-release-packages
 
 main() {
     pushd "$REPO_ROOT" > /dev/null
     git config user.name "${GIT_USER}"
     git config user.email "${GIT_EMAIL}"
 
+    rm -rf $CR_RELEASE_PACKAGES_DIR $CR_INDEX_DIR && mkdir -p $CR_RELEASE_PACKAGES_DIR $CR_INDEX_DIR
```

Comment:
Why are we deleting these folders, and recreating them afterwards? 
If we are only interested in deleting the contents?

Reply:
we can do this too. I just reverted back the old line actually. In any case they are not committed as we excluded them in `.gitignore`.